plugins/org.python.pydev.core/src/org/python/pydev/core/docutils/StringSubstitution.java

```diff
@@ -22,14 +23,18 @@
 import org.eclipse.core.resources.IPathVariableManager;
 import org.eclipse.core.resources.IProject;
 import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.FileLocator;
 import org.eclipse.core.runtime.IPath;
 import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Platform;
 import org.eclipse.core.runtime.Status;
 import org.eclipse.core.variables.VariablesPlugin;
+import org.osgi.framework.Bundle;
 import org.python.pydev.core.IPythonNature;
 import org.python.pydev.core.IPythonPathNature;
 import org.python.pydev.core.log.Log;
-import org.python.pydev.shared_core.string.StringUtils;
+
+import com.sun.tools.javac.util.StringUtils;
```

Comment:
This import shouldn't be changed. We don't want to rely on `com.sun.tools.javac`.